roles/nginx/tasks/main.yml

```diff
@@ -22,10 +22,12 @@
   git:
     repo: "https://github.com/knyar/nginx-lua-prometheus.git"
     dest: /srv/nginx-lua-prometheus
+  tags: monitoring
```

Comment:
registry is deployed with extra parameter --skip-tags monitoring

Reply:
I'm not fan of tags for that purpose, as we will forget to give --skip-tags flag.

May include the nginx part directly in registry task ?

Reply:
we can just install it even if it isn't monitored atm

Reply:
ok